src/main/java/inf112/skeleton/app/screens/GameScreen.java

```diff
@@ -22,6 +24,7 @@ public GameScreen(final RallyGame game) {
 
         this.game = game;
         this.board = game.getBoard();
+        //this.board = menu.getBoard();
```

Comment:
Hvilken av disse skal man bruke?

Reply:
this.board = game.getBoard();
Lekte litt fram, men kommentaren er feil